<?xml version="1.0" ?>
<sdf version="1.5">

    <model name="rock2">
      <static>true</static>
      <link name="rock">
	<visual name="body_visual">
	    <pose>0 0 0 0 0 0</pose>
	    <geometry>
	      <mesh>
		<scale>1.2 1.2 1.2</scale>
		<uri>model://rock2/meshes/rock2.stl</uri>
	      </mesh>
	    </geometry>
	    <material>
	      <script>
		<name>Gazebo/Gray</name>
		<uri>file://media/materials/scripts/gazebo.material</uri>
	      </script>
	    </material>
	  </visual>
      </link>
    </model>

</sdf>
